### PR TITLE
Adds image to classification page

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds a visual to the classification conceptual docs.

### Preview

[Classification docs](https://stack-docs_1868.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-dfa-classification.html)

### CODE REVIEW

File: docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc [hunk 1]
```diff
@@ -13,6 +13,13 @@ classifying malicious and benign domains to detect DGA activities for security
 reasons or predicting customer churn based on customer calling data. 
 {classification-cap} is for predicting discrete, categorical values.
 
+
+image::images/classification-vis.png[alt="Classification process",align="center"]
+The different shapes represent different classes which are devided by the line
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc:18]: In my opinion, we might be able to shorten this, along these lines:

> Classification is a machine learning process that predicts the class or category of a data point in a data set.
> For a simple example, consider how you might learn to differentiate circles from triangles in this graph:
> /image/
> In reality, classification problems are more complex, such as classifying malicious….

Comment (szabosteve) [reply docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc]: @lcawl Updated in 0c59410

File: docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc [hunk 2]
```diff
@@ -8,10 +8,16 @@
 predict classes of data points in a data set.
 
 {classification-cap} is a {ml} process that predicts the class or category of a 
-data point in a data set. Typical examples of {classification} problems are 
-classifying malicious and benign domains to detect DGA activities for security 
-reasons or predicting customer churn based on customer calling data. 
-{classification-cap} is for predicting discrete, categorical values.
+data point in a data set. For a simpe example, consider how the shapes in the
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc:11]:
```suggestion
data point in a data set. For a simple example, consider how the shapes in the 
```

File: docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc [hunk 3]
```diff
@@ -8,10 +8,16 @@
 predict classes of data points in a data set.
 
 {classification-cap} is a {ml} process that predicts the class or category of a 
-data point in a data set. Typical examples of {classification} problems are 
-classifying malicious and benign domains to detect DGA activities for security 
-reasons or predicting customer churn based on customer calling data. 
-{classification-cap} is for predicting discrete, categorical values.
+data point in a data set. For a simpe example, consider how the shapes in the 
+following graph can be differentiated and classified as "circles" and 
+"traingles":
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc:13]:
```suggestion
"triangles":
```

**Review by lcawl — APPROVED** [review]
Two small suggestions, otherwise LGTM!

**Pull request merged**